### ISSUE TITLE
Xplex 1.1 changelog

### ISSUE THREAD

**Issue by RedDragon98** [SpicyDragon95/xplex-issues#22]

**Comment by RedDragon98** [comment]
**Bugs**
Fixed #12 , #6 , #5 as many more bugs. Credits: @Amazingme01 @Spicy-Axolotl @SnowHuskies and AayushAnimalFriends

**Comment by RedDragon98** [comment]
**Dragons**
A new gamemode featuring advanced dragons that try to attack you, breaking blocks as they do that. Be the last one surviving to win! Credits: @Spicy-Axolotl, AayushAnimalFriends

**Comment by RedDragon98** [comment]
**Ore Cube**
Mine ores in a resource rich cube to make better gear and fight others. Credits: @RedDragon98

**Comment by RedDragon98** [comment]
**Trailer**
Credits: @Amazingme01

**Comment by RedDragon98** [comment]
**Sumo**
Punch other players off the platform in this minigame. Credits: the one, ProNoob

**Comment by Spicy-Axolotl** [comment]
> **Dragons** A new gamemode featuring advanced dragons that try to attack you, breaking blocks as they do that. Be the last one surviving to win! Credits: @Spicy-Axolotl, AayushAnimalFriends

Not really a new gamemode.

**Comment by RedDragon98** [comment]
fixed

**Comment by Amazingme01** [comment]
> **Dragons** A new gamemode that has been in development for quite a while featuring advanced dragons that try to attack you, breaking blocks as they do that. Be the last one surviving to win! Credits: @Spicy-Axolotl, AayushAnimalFriends

It's a game, not gamemode

**Comment by Amazingme01** [comment]
Is it tho Sent from Mail for Windows From: RedDragon98Sent: Tuesday, November 22, 2022 4:29 PMTo: SpicyDragon95/xplex-issuesCc: Amazingme01; MentionSubject: Re: [SpicyDragon95/xplex-issues] Xplex 1.1 changelog (Issue #22) fixed—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by Amazingme01** [comment]
> **Dragons** A new gamemode that has been in development for quite a while featuring advanced dragons that try to attack you, breaking blocks as they do that. Be the last one surviving to win! Credits: @Spicy-Axolotl, AayushAnimalFriends

@RedDragon98 It still says gamemode, make it game

**Comment by Amazingme01** [comment]
> also its not a feature, its not in game, just info

right, the text is info, but dragons is a game, not gamemode. Also we should clear this up bc it's a changelog

**Comment by RedDragon98** [comment]
No i'll copy just the real parts later. dont delete comments, also thats fixed

**Comment by poggerslime** [comment]
epic

**Comment by poggerslime** [comment]
> No i'll copy just the real parts later. dont delete comments, also thats fixed

👍

**Comment by poggerslime** [comment]
This is so poggers!!!!!!!!!!

**Comment by Amazingme01** [comment]
> No i'll copy just the real parts later. dont delete comments, also thats fixed

I'm just clearing up the changelog

**Comment by RedDragon98** [comment]
I said dont, theres no point.

**Comment by SnowHuskies** [comment]
@Amazingme01 We don't need to clear up the changelog, its fine. I could read it. Stop trying to cause trouble everywhere please, it's starting to get annoying
